### DISCUSSION TITLE
Add support for xgboost

### DISCUSSION THREAD

**Discussion by GrandioseGuru** [cloudflare/workerd#2141] [Python Packages]
We have some XGBoost models and it'd be excellent to be able to use them in Cloudflare Workers.
XGBoost is one of the most popular Machine Learning libraries that don't use Deep Learning and has no Javascript port, so being able to easily use it would be a nice feature for the Workers platform.
The package is currently included in the stable version of Pyodide.
https://pyodide.org/en/stable/usage/packages-in-pyodide.html
Thanks in advance!

**Comment by arnesund** [comment]
+1
